packages/workspace/src/command-line/format.ts

```diff
@@ -96,17 +96,24 @@ function printError(command: string, e: any) {
 
 function write(patterns: string[]) {
   if (patterns.length > 0) {
-    execSync(`node "${prettierPath()}" --write ${patterns.join(' ')}`, {
-      stdio: [0, 1, 2]
-    });
+    execSync(
+      `node "${prettierPath()}" --write ${patterns
+        .map(pattern => (fileExists(pattern) ? `'${pattern}'` : pattern))
```

Comment:
Why do we need to check if the file exists before wrapping it in quotes? It should be filtered out higher above on https://github.com/nrwl/nx/pull/1485/files#diff-f8549c910ef16fe4ad9fa75fd90c4005R47

Reply:
I originally had it as
```
.map(pattern => `'${pattern}'`
```
but the e2e command-line test failed. This was due to patterns also being wrapped in quotes. The `fileExists` check is to determine whether an element of the patterns input is a pattern e.g. "${prefix}/**/*.{${PRETTIER_EXTENSIONS.join(',')}}" or an actual file. The `fileExists` could be replaced by a `isFile` function that could do some regex to determine whether it s a pattern or a file. What do you think?

Reply:
Right, so we can move this mapping up to https://github.com/nrwl/nx/blob/65ad548ed74f7903575138ed631cd5ff8bc7aacd/packages/workspace/src/command-line/format.ts#L59 which is the only time when it has actual file paths.

Reply:
Oh I see. I'll update the PR soon!